frontend/app/routes/$lang+/_public+/apply+/$id+/tax-filing.tsx

```diff
@@ -83,6 +83,12 @@ export async function action({ context: { session }, params, request }: ActionFu
     return redirect(getPathById('$lang+/_public+/apply+/$id+/file-taxes', params));
   }
 
+  const { typeOfApplication } = await applyRouteHelpers.loadState({ params, request, session });
+
+  if (typeOfApplication === 'personal and child') {
```

Comment:
Should this condition reference the `ApplicantType.Both` enum rather than hardcoding the value?

Reply:
@nicholas-ly no need to export and reuse an enum since the type of `typeOfApplication` is union literal `'delegate' | 'personal' | 'child' | 'personal and child'`. If it would have been of type `string` then yes but not with string literals.